app/controllers/sufia/uploads_controller.rb

```diff
@@ -0,0 +1,8 @@
+class Sufia::UploadsController < ApplicationController
+  before_action :authenticate_user!
```

Comment:
I'm not sure that's required.  They'd already be prohibited from going to the page to create the files.  The worse they could do was upload files that never go to the repository, but we'd know who they are and could talk to them.  I'd say defer taking action until we understand this use case better.